src/vs/editor/browser/viewParts/minimap/minimap.ts

```diff
@@ -680,9 +690,10 @@ export class Minimap extends ViewPart {
 
 	public onThemeChanged(e: viewEvents.ViewThemeChangedEvent): boolean {
 		this._context.model.invalidateMinimapColorCache();
+		this._selectionColor = this._context.theme.getColor(minimapSelection);
 		// Only bother calling render if decorations are currently shown
-		this._renderDecorations = !!this._lastDecorations;
-		return !!this._lastDecorations;
+		this._renderDecorations = !!this._lastDecorations || !!this._selections.length;
```

Comment:
`this._selections.length` is always `>= 1` ... (the editor will always have at least 1 selection), so we might as well say `this._renderDecorations = true;`